manim/utils/config_ops.py

```diff
@@ -56,10 +56,10 @@ def __set_name__(self, obj, name):
         self.name = name
 
     def __get__(self, obj, owner):
-        return obj.__dict__["data"][self.name]
+        return obj.data[self.name] if self.name in obj.data else []
```

Comment:
Checking if an attribute is in `data` might prove to be too expensive and cause a performance dropoff, since `data` is called a lot of times (haven't tested, just speculation). Is there a way to get around doing this?

Reply:
This turned out to be no longer necessary after fixing ugly workarounds described above.